packages/core/package.json

```diff
@@ -74,7 +74,8 @@
   ],
   "sideEffects": false,
   "scripts": {
-    "test": "yarn hardhat-test",
+    "test": "yarn hardhat-test && yarn forge-test",
+    "forge-test":"./scripts/SetUpFoundryWithHardhat.sh && forge test",
```

Comment:
note that this is a no-op if foundry is already setup and installed.

Reply:
Why do we use this script here, but `curl -L https://foundry.paradigm.xyz` above?